ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentationService.java

```diff
@@ -100,17 +79,24 @@ public void handleViewAddedEvent(ViewSupplier event) {
 	private void launchJob(Job job, JobParameters jobParameters) throws JobInstanceAlreadyCompleteException, JobExecutionAlreadyRunningException, JobParametersInvalidException, JobRestartException {
 		JobExecution jobExecution = jobLauncher.run(job, jobParameters);
 
-		if (jobExecution.getStepExecutions().stream().toList().getFirst().getWriteCount() != 0) {
-			if (job.getName().equals(BUCKETISATION_JOB)) {
-				eventPublisher.publishEvent(new MembersBucketisedEvent());
-			} else if (job.getName().equals(REBUCKETISATION_JOB)) {
-				eventPublisher.publishEvent(new NewViewBucketisedEvent(jobParameters.getString("viewName")));
-			}
+		if (jobExecution.getStepExecutions().stream().toList().getFirst().getWriteCount() == 0) {
+			shouldTriggerBucketisation.set(false);
+		}
```

Comment:
Would revert to the previous code in regards to this flag.. this doesn't seem right